### PR TITLE
update the standalone frontend contribs

### PR DESCRIPTION
**Pull request by freben**
Removed the `multi-stage-frontend` one, since it overlapped way too much with the docs, that were more up-to-date anyway.

### CODE REVIEW

**Review by jhaals — APPROVED** [review]

File: contrib/docker/frontend-with-nginx/Dockerfile.dockerbuild [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+#       node_modules
+#       packages/*/dist
+#       packages/*/node_modules
+#       plugins/*/dist
+#       plugins/*/node_modules
+#
+# - Update the copy of this file to add configuration arguments to the "build"
+#   command, for example:
+#
+#       RUN yarn workspace example-app build --config <config1> --config <config2> ...
```
Comment (Rugvip) [inline contrib/docker/frontend-with-nginx/Dockerfile.dockerbuild:30]: Would be just `app` in a create-app'd repo

File: contrib/docker/frontend-with-nginx/Dockerfile.dockerbuild [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+#
+# - In your project root, run:
+#
+#       docker build -t backstage-frontend -f Dockerfile.dockerbuild .
+
+
+
+FROM node:12-buster AS build
```
Comment (Rugvip) [inline contrib/docker/frontend-with-nginx/Dockerfile.dockerbuild:38]: Let's use 14 like the docs, the only ones on 12 are to older contrib ones

File: contrib/docker/frontend-with-nginx/Dockerfile.dockerbuild [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+RUN mkdir /app
+COPY . /app
+WORKDIR /app
+
+RUN yarn install
+RUN yarn workspace example-app build
```
Comment (Rugvip) [inline contrib/docker/frontend-with-nginx/Dockerfile.dockerbuild:45]: Same here with the `app`

**Review by Rugvip — APPROVED** [review]
👍 , just a few fixes

File: contrib/docker/frontend-with-nginx/Dockerfile.dockerbuild (hunk 1 above)
Comment (freben) [reply contrib/docker/frontend-with-nginx/Dockerfile.dockerbuild]: ah right, all fix'd up

**Pull request merged**